### DISCUSSION TITLE
Add OGC Web Services to map canvas

### DISCUSSION THREAD

**Discussion by roycegeo** [apache/superset#26291] [Ideas]
We'd like to request a new map layer type as a standard map chart type. This would include points, lines and polygons as well as imagery all served as standard OGC web services. Web Map Services (WMS), Web Map Tile Service (WMTS) and Web Feature Service (WFS) are all web services that can integrate seamlessly with the MapBox canvas that is in there now, Leaflet and OpenLayers mapping interface. This will allow more layer types to be added to the map chart and will enable unlimited extensibility to the number of data layers viewable in SuperSet.

**Comment by JTvdWal-WUR** [comment]
This will also improve the FOSS-usability of SuperSet in relation to maps.
